docs/releasing.md

```diff
@@ -22,8 +22,17 @@ Creating a federation v2 release involves the following steps:
    1. `make kubefed2`
    2. `cd bin`
    3. `kubefed2 version` (check that the output is as expected)
-   4. `tar cvzf kubefed2.tar.gz kubefed2`
-   5. `sha256sum kubefed2.tar.gz > kubefed2.tar.gz.sha`
+   4. `tar cvzf kubefed2.tgz kubefed2`
+   5. `sha256sum kubefed2.tgz > kubefed2.tgz.sha`
+4. Package the helm chart for release
+   1.  Adjust the default image tag in values.yaml
+   2.  Update the chart version in Chart.yaml
+   3.  `helm package federation-v2` (Package the chart)
+   4.  `sha256sum federation-v2-<x.x.x>.tgz > federation-v2-<x.x.x>.tgz` (Name and checksum the packaged chart accordingly)
+   5.  `helm repo index federation-v2/ --url=https://raw.githubusercontent.com/kubernetes-sigs/federation-v2/add-chart-repo-index/charts/` (Add the new version to the chart index)
```

Comment:
The command should be: `helm repo index .  --merge index.yaml --url=https://github.com/kubernetes-sigs/federation-v2/releases/download/v<x.x.x>/federation-v2-<x.x.x>.tgz`